node/network/bridge/src/multiplexer.rs

```diff
@@ -83,13 +83,13 @@ impl Stream for RequestMultiplexer {
 		// Poll streams in round robin fashion:
 		while count > 0 {
 			// % safe, because count initialized to len, loop would not be entered if 0, also
-            // length of receivers is fixed.
+			// length of receivers is fixed.
 			let (p, rx): &mut (_, _) = &mut self.receivers[i % len];
 			i += 1;
 			count -= 1;
 			match Pin::new(rx).poll_next(cx) {
 				// If at least one stream is pending, then we are not done yet (No
-                // Ready(None)).
+				// Ready(None)).
```

Comment:
At some point, I will get my editor to do this right. It works like 99% of the time, but then suddenly expandtab is activated again for some reason. *arrgh*